style.css

```diff
@@ -0,0 +1,150 @@
+@import "https://fonts.googleapis.com/css?family=Lato:400,700";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #fff;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+
+header {
+  background-color: rgb(255, 255, 255);
+  position: fixed;
+  z-index: 99;
+  top: 0;
+  min-height: 75px;
+  display: flex;
+  padding: 0px 10px;
+  justify-content: space-around;
+  align-items: center;
+  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.19), 0 6px 6px rgba(0, 0, 0, 0.23);
+}
```

Comment:
Kindly consider adjusting the header of the page so it covers all the width, and a proper size to uncover the video below.